### PR TITLE
Update bank.json

### PR DESCRIPTION
**Pull request by rkdclgh**
Update the list of Korean Bank

### CODE REVIEW

File: data/brands/amenity/bank.json [hunk 1]
```diff
@@ -11196,15 +11213,16 @@
       "displayName": "하나은행",
       "id": "koreaexchangebank-10d607",
       "locationSet": {"include": ["kr"]},
+      "matchNames": ["KEB하나은행"],
       "tags": {
         "amenity": "bank",
         "brand": "하나은행",
-        "brand:en": "Korea Exchange Bank",
+        "brand:en": "KEB Hana Bank",
```
Comment (arch0345) [inline data/brands/amenity/bank.json:11220]: According to the [English Wikipedia article](https://en.wikipedia.org/wiki/Hana_Bank), it looks like they're branded in South Korea as just "Hana Bank" in English. Let me know if this is not the case

File: data/brands/amenity/bank.json [hunk 2]
```diff
@@ -11080,62 +11084,73 @@
...
+        "name:en": "NH Bank",
+        "name:ko": "NH농협은행"
       }
     },
     {
       "displayName": "새마을금고",
       "id": "ca214a-10d607",
       "locationSet": {"include": ["kr"]},
+      "matchNames": [
+        "MG새마을금고",
```
Comment (arch0345) [inline data/brands/amenity/bank.json:11151]: Based on [their website](https://www.kfcc.co.kr/) it looks like they brand themselves as "MG새마을금고"

**Review by arch0345 — COMMENTED** [review]
Thanks @rkdclgh for these updates! I've given you some feedback

File: data/brands/amenity/bank.json (hunk 1 above)
Comment (rkdclgh) [reply data/brands/amenity/bank.json:11220]: In general, it is called "Hana Bank", but they have designated the official English name, "KEB Hana Bank".

File: data/brands/amenity/bank.json (hunk 2 above)
Comment (rkdclgh) [reply data/brands/amenity/bank.json]: I updated it. Thanks!

**Pull request merged**